test/chplcheck/RuleSettings/RuleSettings.prediff

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+base=$1
+good=$2
+function testcase() {
+  local testnum=$1
+  shift
+  local args=$@
+  $CHPL_HOME/tools/chplcheck/chplcheck --add-rules rules.py $args $base.chpl > $base.$testnum.out
+  if diff -q $base.$testnum.out $base.$testnum.good; then
+    echo "testcase $testnum passed" >> $good
+    rm $base.$testnum.out
+  else
+    echo "testcase $testnum failed" >> $good
+  fi
```

Comment:
Consider just storing the output of `diff` into the good, so that it's easier to see at a glance without having to re-run chplcheck yourself.